omniscidb/QueryEngine/Descriptors/QueryMemoryDescriptor.cpp

```diff
@@ -1059,8 +1059,13 @@ bool QueryMemoryDescriptor::blocksShareMemory() const {
 }
 
 bool QueryMemoryDescriptor::lazyInitGroups(const ExecutorDeviceType device_type) const {
+#ifdef HAVE_L0
+  return false; /* L0 path currently does not provide lazy group by buffers initialization
```

Comment:
Won't this disable lazy init groups for CUDA too? Or are we not planning to build CUDA + L0?

Reply:
There's no way of building CUDA + L0 currently, so CUDA is not affected.
>Or are we not planning to build CUDA + L0

We do, and there're some changes that minimize the gap we have now, but I have no immediate plans for enabling both in a single build right now. Over time it became much easier to enable, so we might want to do that soon.